build_tools/kickstarts/centos-7-kubernetes-vagrant.ks

```diff
@@ -56,11 +56,7 @@ if [ -b /dev/mapper/vg001-root ]; then
   lvcreate -l 8%FREE -n docker-meta vg001
   lvcreate -l 100%FREE -n docker-data vg001
 
-  cat <<EOF >> /etc/sysconfig/docker-storage
-
-DOCKER_STORAGE_OPTIONS=--storage-opt dm.fs=xfs --storage-opt dm.datadev=/dev/mapper/vg001-docker--data --storage-opt dm.metadatadev=/dev/mapper/vg001-docker--meta
-
-EOF
+  sed -i.back '/DOCKER_STORAGE_OPTIONS=*/c\DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=xfs --storage-opt dm.datadev=/dev/mapper/vg001-docker--data --storage-opt dm.metadatadev=/dev/mapper/vg001-docker--meta"'  /etc/sysconfig/docker-storage
```

Comment:
Use of  dm.datadev and dm.metadatadev is deprecated , use dm.thinpooldev instead. Check https://github.com/docker/docker/blob/master/man/docker.1.md#dmdatadev